<launch>
  <include file="$(find depth_enhance_robot)/launch/panda_bringup.launch"></include>

  <include file="$(find easy_handeye)/launch/calibrate.launch">
    <arg name="eye_on_hand" value="true"/>

    <!-- you can choose any identifier, as long as you use the same for publishing the calibration -->
    <arg name="namespace_prefix" value="my_eih_calib"/>

    <!-- fill in the following parameters according to your robot's published tf frames -->
    <arg name="robot_base_frame" value="/panda_link0"/>
    <arg name="robot_effector_frame" value="/panda_suction_rect"/>
    <arg name="move_group" value="panda_arm"/>

    <!-- fill in the following parameters according to your tracking system's published tf frames -->
    <arg name="tracking_base_frame" value="/camera_color_frame"/>
    <arg name="tracking_marker_frame" value="/optical_target"/>
  </include>
</launch>